tests/test_resources.py

```
@@ -34,55 +26,68 @@ def _mock_ensure_started():
-        _mock_subscribe_once.return_value = {"data": "ok"}
-        mcp = _make_resources()
+class TestLiveResourcesUseManagerCache:
+    """All live resources must read from the persistent SubscriptionManager cache."""
 
-        local_provider = mcp.providers[0]
-        resource_key = f"resource:unraid://live/{action}@"
-        resource = local_provider._components[resource_key]
-        result = await resource.fn()
+    @pytest.mark.parametrize("action", list(SNAPSHOT_ACTIONS.keys()))
+    async def test_resource_returns_cached_data(
+        self, action: str, _mock_ensure_started: AsyncMock
```

Ruff (0.15.5)

[warning] Fixture `_mock_ensure_started` without value is injected as parameter, use `@pytest.mark.usefixtures` instead

(PT019)